cmd/lncli/cmd_payments.go

```diff
@@ -1364,11 +1371,16 @@ func forwardingHistory(ctx *cli.Context) error {
 		maxEvents = uint32(m)
 	}
 
+	if ctx.Bool("skip_peer_alias_lookup") {
+		skipPeerAliasLookup = true
+	}
+
 	req := &lnrpc.ForwardingHistoryRequest{
-		StartTime:    startTime,
-		EndTime:      endTime,
-		IndexOffset:  indexOffset,
-		NumMaxEvents: maxEvents,
+		StartTime:           startTime,
+		EndTime:             endTime,
+		IndexOffset:         indexOffset,
+		NumMaxEvents:        maxEvents,
+		SkipPeerAliasLookup: skipPeerAliasLookup,
```

Comment:
nit: I think you can just set this directly. So: `SkipPeerAliasLookup: ctx.Bool("skip_peer_alias_lookup")`